easybuild/tools/docs.py

```diff
@@ -744,28 +744,65 @@ def list_toolchains_rst(tcs):
     """ Returns overview of all toolchains in rst format """
     title = "List of known toolchains"
 
-    # figure out column names
-    table_titles = ['name', 'compiler', 'MPI']
-    for tc in tcs.values():
-        table_titles.extend(tc.keys())
+    # Specify the column names for the table
+    table_titles = ['name', 'COMPILER', 'MPI', 'linalg', 'FFT']
 
+    # Set up column name : display name pairs
     col_names = {
-        'COMPILER_CUDA': 'CUDA compiler',
-        'SCALAPACK': 'ScaLAPACK',
+        'name': 'Name',
+        'compiler': 'Compiler(s)',
+        'linalg': "Linear algebra",
     }
 
-    table_titles = nub(table_titles)
+    # Create sorted list of toolchain names
+    sorted_tc_names = sorted(tcs.keys(), key=str.lower)
 
-    table_values = [[] for i in range(len(table_titles))]
-    table_values[0] = ['**%s**' % tcname for tcname in tcs.keys()]
+    # Create text placeholder to use for missing entries.
+    # Not good documentation style to italicize the parentheses enclosing
+    # italic text unless surrouning text is also italic.
+    none_txt = '(*none*)'
```

Comment:
also making the parentheses italic was deliberate actually, I think it looks awkward otherwise...

compare '*(none)*' (fully italic) with '(*none*)' (partial italic)

anyway, this is [bikeshedding](https://en.wikipedia.org/wiki/Law_of_triviality), definitely not a blocker for this PR :)

Reply:
I was trained as a typesetter and spent years as a proofreader and (English) copy editor, and I have the opposite reaction to _(none)_ and (_none_)
https://english.stackexchange.com/questions/9878/should-punctuation-surrounding-italicised-words-be-italicised
But, this is your baby, so I will do it your way.

Reply:
Oh boy, please don't agree with me if you're better trained than me in this stuff! :)

I was just going by feeling, if there are clear arguments against using `*(none)*`, then please don't!

Reply:
If it would lead to inconsistencies elsewhere, it's fine.  It's just a convention.  ;-)  Just trying to explain why my instincts are what they are.  And that it was not entirely idiosyncratic.  ;-)

I suspect that most everyone else on the project would do what you did and put the *s around the parentheses, and it's probably more important to be consistent throughout.  I changed them, so let's not change back and get this out the door.   :-)